ext/Client/__init__.lua

```diff
@@ -288,32 +303,39 @@ function VEManagerClient:OnVEGuidRequest(p_ID)
 	end
 end
 
----@param p_ID string|nil
+---@param p_ID string
 function VEManagerClient:OnVEDestroyRequest(p_ID)
 	if not self:CheckPresetID(p_ID) then return end
 
 	self.m_Presets[p_ID].ve = nil
 	self.m_Presets[p_ID].entity:Destroy()
 end
 
----@param p_ID string|nil
----@param p_Replacement string|nil
```

Comment:
it is a json string

Reply:
(because u changed it to table)

Reply:
As far as I see m_RawPresets hold lua tables from the decoded json, so if p_Replacement is a string this is missing a json.parse